.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: elixir
 elixir:
    - 1.6.3
-script: mix test
-
+script: MIX_ENV=test mix test
```

Comment:
I think in Travis you use the `env` header to set environment variables: https://docs.travis-ci.com/user/environment-variables/